avogadro/qtplugins/plugindownloader/plugindownloader.h

```diff
@@ -0,0 +1,56 @@
+#ifndef AVOGADRO_QTPLUGINS_PLUGINDOWNLOADER_H
+#define AVOGADRO_QTPLUGINS_PLUGINDOWNLOADER_H
+
+#include <avogadro/qtgui/extensionplugin.h>
+
+#include <avogadro/core/avogadrocore.h>
+
+#include <QtCore/QString>
+
+class QNetworkAccessManager;
+class QNetworkReply;
+class QProgressDialog;
+
+namespace Avogadro {
+namespace QtPlugins {
+
+/**
+ * @brief Downlaods Github repos and extracts their contents into a Avogadro
+ * folder for plugins, molecule data, etc..
+ */
+
+class PluginDownloader : public QtGui::ExtensionPlugin {
+  Q_OBJECT
+
+ public:
```

Comment:
Same here, no indent for public: etc, and opening { for classes, functions etc on their own line.